assets/js/components/user-input/UserInputKeywords.js

```diff
@@ -120,8 +120,16 @@ export default function UserInputKeywords( { slug, max } ) {
 							'googlesitekit-user-input__text-option': values.length > i + 1 || value.length > 0,
 						} ) }
 					>
+						<label
+							htmlFor={ `${ slug }-keyword-${ i }` }
+							className="screen-reader-text"
+						>
+							{ __( 'Enter minimum one (1), maximum three (3) terms', 'google-site-kit' ) }
+						</label>
```

Comment:
@Hazlitte let's move this label out of the `values.map` callback because currently, it generates three identical labels with the same text which might confuse users that use screen readers. Just move it above the `values.map` section and I believe it should be good to go.

Reply:
@eugene-manuilov Ok.